### DISCUSSION TITLE
ManagedBookmarks not picking up on mac with plist

### DISCUSSION THREAD

**Discussion by sstryhalaleck** [mozilla/policy-templates#1073] [Q&A]
Hi there,

I am trying to use the macOS plist to deploy company bookmarks with Jamf. The other settings in the file are picked up just fine. I have tried just using the example of the policy template page, and nothing. I know on the Windows side, you have to use JSON, and that worked pretty easily. Is Firefox on Mac expecting to see the policy as JSON as well?

**Comment by mkaply** [comment]
Do you see any errors when you go to about:policies?
